pylutron/__init__.py

```diff
@@ -298,15 +320,26 @@ def _parse_area(self, area_xml):
             'HYBRID_SEETOUCH_KEYPAD',
             'MAIN_REPEATER',
             'HOMEOWNER_KEYPAD',
-            'GRAFIK_T_HYBRID_KEYPAD'):
+            'INTERNATIONAL_SEETOUCH_KEYPAD',
+            'WCI',
+            'QS_IO_INTERFACE',
+            'GRAFIK_T_HYBRID_KEYPAD',
+            'HWI_SLIM'
+        ):
+
           keypad = self._parse_keypad(device_xml, device_group)
           area.add_keypad(keypad)
         elif device_xml.get('DeviceType') == 'MOTION_SENSOR':
           motion_sensor = self._parse_motion_sensor(device_xml)
           area.add_sensor(motion_sensor)
         #elif device_xml.get('DeviceType') == 'VISOR_CONTROL_RECEIVER':
+        else:
+          #phantom keypad doesn't have a DeviceType
```

Comment:
I think about here is where the logic for a phantom keypad should live.  Check if it's CSD 001, and if so, call self._parse_phantom_keypad(...).  Put in the custom logic for the device type, etc... there.

Reply:
"CSD 001" is used for example for INTERNATIONAL_SEETOUCH_KEYPAD. Note that I've no UUID in any component. I don't know if this due to the Homeworks QS or it's version.

Phantom keypada are not the CDS 001 keypads, but the one that you can create in the integration

Reply:
@cdheiser can we resolve this?

Reply:
ok... I'm just trying to follow along and make suggestions on how to get some custom handling out of the keypad class.  I'd like it to not care about odd variants.  Odd variants should either be prepped outside the class initialization, or as their own distinct type.

Reply:
I've moved the device_type before the the parse_keypad